torch_xla/utils/gcsfs.py

```diff
@@ -214,3 +216,35 @@ def write(path, content):
   gcs_file = torch_xla._XLAC._xla_tffile_create(path)
   torch_xla._XLAC._xla_tffile_write(gcs_file, content)
   torch_xla._XLAC._xla_tffile_flush(gcs_file)
+
+
+def generic_write(output_string, output_path):
+  """Write a string/bytes or file into a GCS blob or local disk, depending
+  on the output_path passed in. Checks if the `output_path` starts with
+  the 'gs://' prefix, and uses `open` otherwise.
+
+  Args:
+    output_string (string): The string to be written to the output.
+    output_path (string): The GCS path or local path of the output.
+  """
+  if output_path.startswith(CLOUD_STORAGE_PREFIX):
+    write(output_path, output_string)
+  else:
+    with open(output_path, 'w') as outfile:
```

Comment:
Lets use `fd` here and below as open() context.

Reply:
do you mean `s/outfile/fd/` ?

Reply:
Yeah